tests/filecheck/dialects/tensor/ops.mlir

```diff
@@ -1,24 +1,31 @@
 // RUN: XDSL_ROUNDTRIP
 // RUN: XDSL_GENERIC_ROUNDTRIP
 
-%index, %tensor = "test.op"() : () -> (index, tensor<?x?xf32>)
+%index, %index1, %tensor = "test.op"() : () -> (index, index, tensor<?x?xf32>)
 %dim1 = "tensor.dim"(%tensor, %index): (tensor<?x?xf32>, index) -> index
 %dim2 = "tensor.dim"(%tensor, %index) {"hello" = "world"}: (tensor<?x?xf32>, index) -> index
 %cast1 = "tensor.cast"(%tensor) : (tensor<?x?xf32>) -> tensor<4x4xf32>
 %cast2 = "tensor.cast"(%tensor) {"hello" = "world"} : (tensor<?x?xf32>) -> tensor<4x4xf32>
+%extract1 = "tensor.extract"(%tensor, %index, %index1) : (tensor<?x?xf32>, index, index) -> f32
+%insert1 = "tensor.insert"(%extract1, %tensor, %index, %index1) : (f32, tensor<?x?xf32>, index, index) -> tensor<?x?xf32>
```

Comment:
While you're making changes here, could you please rewrite the inputs in custom syntax? Will be a little easier to read for the next person